lib/core/rollup.config.js

```diff
@@ -0,0 +1,280 @@
+import fs from "fs";
+import path from "path";
+import vue from "rollup-plugin-vue";
+import alias from "@rollup/plugin-alias";
+import commonjs from "@rollup/plugin-commonjs";
+import resolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import babel from "@rollup/plugin-babel";
+import PostCSS from "rollup-plugin-postcss";
```

Comment:
@tuhuynh27 @monodyle is it possible to NOT having PostCSS at first? I mean we will eventually need it, but setting it up is not simple due to the huge breaking change of PostCSS 8.

You can use the "postcss-combat" solution, but it would significantly increase the complexity of this PR.. so my suggestion is to just remove it for now

Reply:
Okay